core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -101,4 +101,16 @@ oppia.controller('CollectionEditor', ['$scope',
             'There was an error when publishing the collection.');
         });
     };
+
+    // Unpublish the collection. Will only show up if the collection is public.
```

Comment:
This should also only show up if the user has access to unpublish a collection. You will need to send some sort of variable containing this, like:

https://github.com/oppia/oppia/blob/dcd59cab0cf8aadeb515d0882be8d8fba6f4c9eb/core/controllers/editor.py#L228

Then you can save it to the ``GLOBALS`` object similar to:

https://github.com/oppia/oppia/blob/dcd59cab0cf8aadeb515d0882be8d8fba6f4c9eb/core/templates/dev/head/collection_editor/collection_editor.html#L15